src/experimentalOmnisharp.Download.Helper.ts

```diff
@@ -0,0 +1,82 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+import * as vscode from 'vscode';
+import { PackageManager, Status, PackageError, Package } from './packages';
+import { PlatformInformation } from './platform';
+import { Logger } from './logger';
+import TelemetryReporter from 'vscode-extension-telemetry';
+
+export async function GetDependenciesAndDownloadPackages(packages: Package[],status: Status, platformInfo: PlatformInformation, packageManager: PackageManager,logger: Logger) { 
+    const config = vscode.workspace.getConfiguration();
+    const proxy = config.get<string>('http.proxy');
+    const strictSSL = config.get('http.proxyStrictSSL', true);
+    await packageManager.DownloadPackages(logger, status, proxy, strictSSL);
```

Comment:
How do other callers do this setup before calling download packages? Should proxy configuration just be a feature of DownloadPackages?

Reply:
There is only one caller - the CSharpExtDownloader other than the OmnisharpDownloader and I have configured it the same way. There might be other ways to set proxy and strictSSL so moving this code to the DownloadPackages may not be a good idea. Thoughts ?